communication/meeting-notes-archive/201902-201911_Community_Meeting_Notes.md

```diff
@@ -2325,6 +2325,6 @@ Twitter | [Sep 19th](https://twitter.com/stephenaugustus/status/1174797710043430
         *   bentheelder - Shoutout to @jeefy for the really shiny and useful looking structured Kubernetes release notes viewer demo in today's #sig-release meeting, looking forward to seeing more about this! 
         *   Maria - shoutout to Silvia Moura Pina (@smourapina) for putting together a workflow to enable the CI signal subteam of the release team keep on top of flagging issues from e2e tests and coordinate follow-ups and to Jorge Alarcon (@Jorge) for spotting an opportunity to offer broader transparency to what the CI signal team is working on, suggesting a structure and kicking off implementation (find current version at [https://github.com/orgs/kubernetes/projects/11](https://github.com/orgs/kubernetes/projects/11))
         *   spiffxp - shouts to Josh Berkus (@jberkus) for taking notes during today’s steering committee meeting, our google doc clearly becomes way more difficult to use with so many people looking at it, and the written record is invaluable
-        *   spiffxp - shoutout to Katharine Berry (@Katharine) for moving us from gubernator to spyglass for all of our test result viewing needs! [https://git.k8s.io/test-infra/prow/spyglass ](https://git.k8s.io/test-infra/prow/spyglass)
+        *   spiffxp - shoutout to Katharine Berry (@Katharine) for moving us from gubernator to spyglass for all of our test result viewing needs! [http://sigs.k8s.io/prow/pkg/spyglass ](http://sigs.k8s.io/prow/pkg/spyglass)
```

Comment:
@Okabe-Junya, minor nit for all url changes.

all `sigs.k8s.io` links should be using `https://sigs.k8s.io/...`.

Example below. Similar need to be changed for all links.

```suggestion
        *   spiffxp - shoutout to Katharine Berry (@Katharine) for moving us from gubernator to spyglass for all of our test result viewing needs! [https://sigs.k8s.io/prow/pkg/spyglass ](https://sigs.k8s.io/prow/pkg/spyglass)
```

Reply:
Thanks!! fixed in https://github.com/kubernetes/community/pull/7884/commits/0adfecd04da83558fea14f42869ab8f12dc062bb